Doc/library/datetime.rst

```diff
@@ -1786,7 +1786,7 @@ Other constructor:
        datetime.time(4, 23, 1)
        >>> time.fromisoformat('04:23:01.000384')
        datetime.time(4, 23, 1, 384)
-       >>> time.fromisoformat('04:23:01,000')
+       >>> time.fromisoformat('04:23:01,000384')
```

Comment:
There was only a little information for me to interpret the intention of the code example. Given that the returned value of the previous line was the same as this line, I can only guess that this line was meant to show that the comma is strictly equivalent to the dot as the delimiter for fractional seconds. Therefore, I decided to change the returned value of this line.

I think it would be more clear if there were separate examples of showing the equivalence of delimiters and showing the difference between two fractional seconds formats. If this was the case, I would like to keep the current change and add one more code example to demonstrate the 3-digits format for fractional seconds. For example:

```python
>>> time.fromisoformat('04:23:01.384')
datetime.time(4, 23, 1, 384000)
```

Reply:
`time.fromisoformat('04:23:01.384')` was supported in 3.10, but neither `time.fromisoformat('04:23:01,000384')` nor `time.fromisoformat('04:23:01,384')` worked. So this new test was for comma, not for three-digits fraction part.